LDM-732.tex

```diff
@@ -60,35 +55,22 @@ \section{Introduction}\label{sec:intro}
 
 \subsection{Scope}\label{sec:scope}
 
-This plan governs only tests of the network infrastructure, not the applications and services that
-use the network. To be specific, this plan governs tests of the network only up to ISO OSI Layer
-\url{https://en.wikipedia.org/wiki/OSI_model}.
+This plan governs only tests of the network infrastructure, not the applications and services that use the network. To be specific, this plan governs tests of the network only up to \href{https://en.wikipedia.org/wiki/OSI_model}{OSI Layer} 3.
```

Comment:
This document is not a test plan it is a verification document.  => This  document pertains only to the verification of the network infrastructure, not the applications and services that use the network, specifically, up to \href{https://en.wikipedia.org/wiki/OSI_model}{OSI Layer} 3.